overlord/configstate/configcore/runwithstate.go

```diff
@@ -38,9 +38,11 @@ func init() {
 
 	// proxy.{http,https,ftp}
 	addWithStateHandler(validateProxyStore, handleProxyConfiguration, coreOnly)
-	addWithStateHandler(validateRefreshSchedule, nil, nil)
-	addWithStateHandler(validateRefreshRateLimit, nil, nil)
-	addWithStateHandler(validateAutomaticSnapshotsExpiration, nil, nil)
+
+	validateOnly := &flags{coreOnlyConfig: true, validatedOnlyStateConfig: true}
```

Comment:
this was not coreOnly no?

Reply:
Yes, restored, thanks. As discussed this is irrelevant (didn't fail any test) because validate is unconditional (doesn't care about coreOnlyConfig flag; only `handle` cares.